### DISCUSSION TITLE
Sequelize > 6.0 Require and Import in ES6

### DISCUSSION THREAD

**Discussion by GonzaloSP** [sequelize/sequelize#13391] [Help & Questions] [answered]
Hi, 

I am using Sequelize 6, and the default way Sequelize generates the importing of the modules is:

`const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes)
`

The issue is that because I am using ES6 (I defined in package.json as type=module) I cannot use require as it is not defined. The other way of importing the models is with:

`sequelize['import']
`

But this function has been deprecated in Sequelize 6.0.

What's the way of loading all the models in ES6?

Thanks

**Comment by ephys** [comment, marked as answer]
Relevant discussions: https://github.com/sequelize/sequelize/issues/11777 & https://github.com/sequelize/sequelize/issues/13132

if you're using ESM, you should have access to `import`, instead of doing

```
const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes) 
```

simply do

```
import model from './path-to-model.js';
```
